commons-vfs2/src/main/java/org/apache/commons/vfs2/cache/SoftRefFilesCache.java

```diff
@@ -290,6 +286,9 @@ private boolean removeFile(final FileSystemAndNameKey key) {
         return files.isEmpty();
     }
 
+    /**
+     * Called while the lock is held
+     */
     protected Map<FileName, Reference<FileObject>> getOrCreateFilesystemCache(final FileSystem fileSystem) {
```

Comment:
Could you make this `private` so that it's consistent with all the other methods that are "called while the lock is held"?